### PR TITLE
[zh]translate content/docs/reference/kubernetes-api/common-definition…

### PR DESCRIPTION
**Pull request by howieyuen**
related: https://github.com/kubernetes/website/issues/32435

### CODE REVIEW

File: content/zh/docs/reference/kubernetes-api/common-definitions/status.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    *StatusCause 提供有关 api.Status 失败的更多信息，包括遇到多个错误的情况。*
+
+    - **details.causes.field** (string)
+
+      <!-- 
+      The field of the resource that has caused this error, as named by its JSON serialization. 
+      May include dot and postfix notation for nested attributes. Arrays are zero-indexed.  
+      Fields may appear more than once in an array of causes due to fields having multiple errors. Optional.
+      -->
+      导致此错误的资源字段，由其 JSON 序列化命名。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/status.md:103]:
```suggestion
      导致此错误的资源字段，根据其 JSON 序列化命名。
```

**Review by tengqm — COMMENTED** [review]
this is a dup?

**Comment by tengqm** [comment]
https://github.com/kubernetes/website/pull/32197

**Comment by howieyuen** [comment]
> #32197

oops...I lost my sight of this PR, I close this first, reopen anytime if needed

**Comment by howieyuen** [comment]
> > #32197
> 
> oops...I lost my sight of this PR, I close this first, reopen anytime if needed

reopen this one since #32197 is closed, because it's inactive over 3 weeks

**Pull request merged**